pkg/test/framework/components/istio/eastwest.go

```diff
@@ -86,6 +86,15 @@ func (i *operatorComponent) deployEastWestGateway(cluster resource.Cluster, revi
 		installSettings = append(installSettings, "--revision", revision)
 	}
 
+	// Save the manifest generate output so we can later cleanup
+	genCmd := []string{"manifest", "generate"}
```

Comment:
we did not save eastwest deploy. This copies the code we use for the main install